crates/block-producer/README.md

```diff
@@ -49,5 +49,10 @@ can know for certain it was evicted.
 
 ---
 
+## Crate Features
+
+- `tracing-forest`: sets logging using tracing-forest. Disabled by default.
+- `testing`: includes testing util functions to mock block-producer behaviour, meant to be used during development and not in production. Disabled by default.
```

Comment:
Is this still correct? I don't think there is a `testing` feature currently for the block producer.